### PR TITLE
[ci skip] Fix hoglin conversionTime javadoc

### PR DESCRIPTION
**Pull request by Lulu13022002**
That's some stuff i have forgot in my last pr. I'm not really sure if this patch shouldn't be merged with the upstream patch

### CODE REVIEW

**Review by Owen1212055 — APPROVED** [review]

File: patches/api/0336-Fix-issues-with-mob-conversion.patch [hunk 1]
```diff
@@ -4,6 +4,33 @@ Date: Sun, 24 Oct 2021 20:29:27 -0700
...
+index b49670816d0d42aee08614d9a59a1f785ad8e6bc..8e5477081f56a7d27415da099237e8a1507318b4 100644
+--- a/src/main/java/org/bukkit/entity/Hoglin.java
++++ b/src/main/java/org/bukkit/entity/Hoglin.java
+@@ -44,13 +44,17 @@ public interface Hoglin extends Animals {
+     public int getConversionTime();
+ 
+     /**
+-     * Sets the amount of ticks until this entity will be converted to a Zoglin.
++     * Sets the conversion counter value. The counter is incremented
++     * every tick the {@link #isConverting()} returns true. Setting this
```
Comment (lynxplay) [inline patches/api/0336-Fix-issues-with-mob-conversion.patch]:
```suggestion
+     * every tick the method {@link #isConverting()} returns true. Setting this
```

File: patches/api/0336-Fix-issues-with-mob-conversion.patch [hunk 2]
```diff
@@ -4,6 +4,33 @@ Date: Sun, 24 Oct 2021 20:29:27 -0700
...
++     * value will not start the conversion if the {@link Hoglin} is
++     * not in a valid environment ({@link org.bukkit.World#isPiglinSafe})
++     * to convert or {@link #isImmuneToZombification()} is true or
++     * has no AI.
```
Comment (lynxplay) [inline patches/api/0336-Fix-issues-with-mob-conversion.patch]:
```suggestion
+     * not in a valid environment ({@link org.bukkit.World#isPiglinSafe})
+     * to convert, is immune to zombification ({@link #isImmuneToZombification()}) 
+     * has no ai ({@link #hasAI}).
```

**Review by lynxplay — APPROVED** [review]
LGTM. Proposed an alternative structure to a few sentences, but mere suggestions, this also works 👍

File: patches/api/0336-Fix-issues-with-mob-conversion.patch (hunk 2 above)
Comment (Lulu13022002) [reply patches/api/0336-Fix-issues-with-mob-conversion.patch]: Yes it's better thanks

**Review by Machine-Maker — APPROVED** [review]

**Review by kennytv — APPROVED** [review]

**Pull request merged**